services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -101,7 +101,7 @@ createUser suid (Id buid) teamid = do
     . path "/i/users"
     . json newUser
     . expect2xx
-  parseResponse resp
+  userId . selfUser <$> parseResponse @SelfProfile resp
```

Comment:
`Spar.Intra.Brig.createUser` seems unused in any tests. How about adding this call to a test case?